neuralmonkey/model/sequence.py

```diff
@@ -151,7 +151,8 @@ def embedding_matrices(self) -> List[tf.Tensor]:
             get_variable(
                 name="embedding_matrix_{}".format(i),
                 shape=[vocab_size, emb_size],
-                initializer=tf.glorot_uniform_initializer())
+                initializer=tf.variance_scaling_initializer(
```

Comment:
zde bych misto glorota dal normalni rozdeleni se std=0.001 ... experimentoval jsem s embedingama loni  "An Exploration of Word Embedding Initialization in Deep-Learning Tasks" ... glorot na embedingach neni dobry napad jelikoz dela inicializaci podle velikosti embedingu a velikosti slovniku, coz je blbost. Napriklad pro slovnik 80k a embeding 500 bude delat uniformni s maximem sqrt(6/40250)=0.012 ... a pro slovnik 30k to bude uniformni sqrt(6/15250)=0.02 ... pri prevodu na normalni to v druhem pripade bude std=0.013.
V paperu, ktery jsem delal loni na inicializaci embedingu napric ruznyma ulohama jsem dosel k tomu, ze cim mensi inicializace tim lepe pro embedingy (dokonce i inicializace nulama fungovala krasne). Proto bych zde zadratoval Normal(std=0.001). 
Kdyby zde byl fan_out, ktery uvazuje pouze velikost embedingu a nikoli slovnik a cimz u embedingu velikosti 500 bude odpovidat Normal(std=0.072) tak to take bude horsi nez dat napevno nejakou opravdu malou odchylky

Reply:
Dík, dávám tam teda to normální (předtím jsme tam měli uniform od -0.5 do 0.5)